### ISSUE TITLE
gardening: should all files start capitalized?

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/zt-app#45]
Some do, some don't. Do we have a rule? Do we want one?

**Comment by khamui** [comment]
yes true. The constants files are starting lower case currently. All other JS files (components, containers, helpers/services) are capitalized. So would be probably more consistent to change it to `Colors.js`, `Fonts.js` and `Spacings.js`?
